docs/release/trg-8/trg-8-00.md

```diff
@@ -0,0 +1,66 @@
+---
+title: TRG 8.00 - Security Scanning Toolchain
+---
+
+| Status | Created     | Post-History                         |
+|--------|-------------|--------------------------------------|
+| Active | 21-Feb-2024 | Initial release                      |
+
+## Why
+
+Our primary aim is to improve security and define best practices across the Tractus-X ecosystem.
+
+## Description
+
+A security scanning toolchain is a collection of tools and processes that are used to scan software applications for security vulnerabilities. These tools can be used at various stages of the software development lifecycle (**SDLC**), from development to deployment and beyond.
+
+:::caution
+
+To pass the quality gates, all **critical** and **high** security vulnerabilities **must be mitigated**.

```

Comment:
Maybe add a note that this generic statement can be in contradiction with [TRG 4.02 use base images as is](https://eclipse-tractusx.github.io/docs/release/trg-4/trg-4-02#use-base-image-as-is) when it comes to containers scans and that the statement from TRG 4.02 prevails.

Reply:
@evegufy thank you for all your feeback, I will make changes according to your suggestions